### ISSUE TITLE
[iOS] sharedElementTransition

### ISSUE THREAD

**Issue by garydevenay** [wix/react-native-navigation#6141]
### Issue Description
Shared element transition throws `NSInvalidArgumentException` on push

`2020-04-20 12:55:58.890768+0100 satc[74909:1650010] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[UILayoutContainerView bridge]: unrecognized selector sent to instance 0x7fd94f21d230'`

### Steps to Reproduce / Code Snippets / Screenshots

```js
	const onSearchBarPress = () => {
		Navigation.push(props.componentId, {
			component: {
				name: 'navigation.Search',
				passProps: {
					id: id
				},
				options: {
					animations: {
						push: {
							content: {
								translationY: {
									from: height,
									to: 0,
									duration: 300
								},
								alpha: {
									from: 1,
									to: 1
								}
							},
							sharedElementTransitions:[
								{
									fromId: 'SearchBoxDest',
									toId: 'SearchBox',
									interpolation: 'linear'
								}
							]
						}
					}
				}
			}
		});
	}

	return (
		<SafeAreaView style={styles.container}>
			<View style={styles.searchContainer} nativeID="SearchBox">
				<TouchableOpacity style={styles.searchBar} onPress={() => onSearchBarPress()}>
					<Icon name="search" size={20} color="#221f1f" style={styles.icon} />
					<Text style={styles.placeHolder}>Where to go?</Text>
				</TouchableOpacity>
			</View>
		</SafeAreaView>
	)
```

```js
    return (
        <SafeAreaView style={styles.container} nativeID="SearchBoxDest">
            <View style={styles.search}>
                <Text style={styles.placeHolder}>Where to go?</Text>
            </View>
        </SafeAreaView>
    )
```
---
### Environment
* React Native Navigation version: 6.4.0
* React Native version: 0.62.2
* Platform(s) (iOS, Android, or both?): iOS (Not checked Android)
* Device info (Simulator/Device? OS version? Debug/Release?): Simulator, Debug, iOS 13.4.1

**Comment by guyca** [comment]
Hey @garydevenay 

Any chance the id's used are mixed up? Shouldn't `fromId` be `SearchBox` and `toId` be `SearchBoxDest`?

Also, worth trying to remove the content animation as shared element transition uses a cross fade animation by default. The `fade` node in the animation you've declared is missing `duration` perhaps that causes the issue

**Comment by garydevenay** [comment]
> Hey @garydevenay
> 
> Any chance the id's used are mixed up? Shouldn't `fromId` be `SearchBox` and `toId` be `SearchBoxDest`?
> 
> Also, worth trying to remove the content animation as shared element transition uses a cross fade animation by default. The `fade` node in the animation you've declared is missing `duration` perhaps that causes the issue

I actually just switched these around to try (incase it was counter intuative). I can confirm I get the same error no matter what the IDs are.

Also tried with no other animations except the sharedElementTransition, but no luck!

**Comment by garydevenay** [comment]
Forgot to mention you in the reply @guyca

**Comment by guyca** [comment]
no worries, what do you mean "switched these around" ? Have you tried:

```
fromId: 'SearchBox',
toId: 'SearchBoxDest'
```

I mean, the id with the `dest` suffix should be to `toId`

**Comment by garydevenay** [comment]
@guyca Sorry for being unclear. Yes my original implementation had
```js
fromId: 'SearchBox',
toId: 'SearchBoxDest'
```
I tried switching the IDs before posting the question just to see if I was misunderstanding how things worked. I can confirm that with SearchBox as `fromId` and SearchBoxDest as `toId` I get the same error.

I have tried removing all other animations from the transition and also filling out all properties for the sharedElementTransition:

```js
sharedElementTransitions:[
{
    fromId: 'SearchBox',
    toId: 'SearchBoxDest',
    interpolation: 'linear',
    duration: 300
}
]
```

Still seem to be getting the above error.
